jdbc/src/main/java/io/kestra/jdbc/repository/AbstractJdbcDashboardRepository.java

```diff
@@ -129,11 +134,18 @@ public Dashboard delete(String tenantId, String id) {
 
     @Override
     public <F extends Enum<F>> ArrayListTotal<Map<String, Object>> generate(String tenantId, DataChart<?, DataFilter<F, ? extends ColumnDescriptor<F>>> dataChart, ZonedDateTime startDate, ZonedDateTime endDate, Pageable pageable) throws IOException {
-        throw new NotImplementedException();
+        Map<Class<? extends QueryBuilderInterface<?>>, QueryBuilderInterface<?>> queryBuilderByHandledFields = new ConcurrentHashMap<>();
```

Comment:
```suggestion
        Map<Class<? extends QueryBuilderInterface<?>>, QueryBuilderInterface<?>> queryBuilderByHandledFields = new HashMap<>();
```

`ConcurrentHashMap` is only useful when used concurrently by multiple thread which is not the case here.